.tekton/numaresources-operator-4-20-pull-request.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  annotations:
+    build.appstudio.openshift.io/repo: https://github.com/openshift-kni/numaresources-operator?rev={{revision}}
+    build.appstudio.redhat.com/commit_sha: '{{revision}}'
+    build.appstudio.redhat.com/pull_request_number: '{{pull_request_number}}'
+    build.appstudio.redhat.com/target_branch: '{{target_branch}}'
+    pipelinesascode.tekton.dev/cancel-in-progress: "true"
+    pipelinesascode.tekton.dev/max-keep-runs: "3"
+    pipelinesascode.tekton.dev/on-cel-expression: |
+      event == "pull_request" &&
+      target_branch == "main" &&
+      (
+        '.tekton/build-pipeline.yaml'.pathChanged() ||
+        '.konflux/container-build.args'.pathChanged() ||
+        'api/***'.pathChanged() ||
+        'cmd/***'.pathChanged() ||
+        'config/***'.pathChanged() ||
+        'hack/***'.pathChanged() ||
+        'internal/***'.pathChanged() ||
+        'must-gather/***'.pathChanged() ||        
+        'nrovalidate/***'.pathChanged() ||
+        'pkg/***'.pathChanged() ||
+        'rte/***'.pathChanged() ||
+        'tools/***'.pathChanged() ||
+        'vendor/***'.pathChanged() ||
+        'go.mod'.pathChanged() ||
+        'go.sum'.pathChanged() ||
+        '.tekton/numaresources-operator-4-20-pull-request.yaml'.pathChanged() ||
+        '.konflux/operator/konflux.Dockerfile'.pathChanged()
```

Comment:
I think you want the operator to build if `.konflux/operator` as a whole changes (rpm changes too)